### PR TITLE
438 merge balances with same coin

### PR DESCRIPTION
**Pull request by ladovina**

### CODE REVIEW

File: blockapi/v2/models.py [hunk 1]
```diff
@@ -246,6 +246,21 @@ def from_api(
             is_wallet=is_wallet,
         )
 
+    def __add__(self, other):
+        return BalanceItem(
+            balance_raw=self.balance_raw + other.balance_raw,
+            balance=self.balance + other.balance,
+            coin=self.coin,
+            asset_type=self.asset_type,
+            raw={
```
Comment (ladovina) [inline blockapi/v2/models.py:255]: TODO: this has to be changed since currently both adresses are the same ...

File: blockapi/v2/api/solana.py [hunk 2]
```diff
@@ -146,3 +169,14 @@ def _opt_raise_on_other_error(self, response: Response) -> None:
...
+
+if __name__ == "__main__":
+    from blockapi.v2.api.solana import SolanaApi
+
+    balances = SolanaApi().get_balance('FEeSRuEDk8ENZbpzXjn4uHPz3LQijbeKRzhqVr5zPSJ9')
+
+    y = []
+    for item in balances:
+        if item.coin.address.lower() == 'hel6kguevwygttcjenf9qeab2zg9yr77uswpy9uzvoqj':
+            y.append(item)
```
Comment (galvanizze) [inline blockapi/v2/api/solana.py:182]: this is for testing purposes?

File: blockapi/v2/models.py (hunk 1 above)
Comment (galvanizze) [reply blockapi/v2/models.py:255]: maybe do it as list?

Comment (ladovina) [reply blockapi/v2/models.py]: resolved offline - dict with "merged" keyword containing list of the all of merged `raw` items

File: blockapi/v2/api/solana.py (hunk 2 above)
Comment (ladovina) [reply blockapi/v2/api/solana.py:182]: Yes - it was before I had the unit test ... removed :+1:

**Review by galvanizze — APPROVED** [review]

**Pull request merged**